Brisanet/ion
# 447 popover in indicator component

## Issue Number

fix #447 

## Description

The indicator component should have a option to open a popover, so I added a button with the popover directive that will be rendered when the type popover is informed, and also implemented the popover input properties on the indicator template so the indicator can pass it to the popover.

## Proposed Changes

- Changed the logic of the HTML, so the button will be conditionally rendered; 
- changed the indicator type, so it can receive the popover props and the actions callbacks;
- created a new mock component and a story to visualise the implementation; 
- implemented new test assertions to ensure that the rendering will work as expected.

## How to Test

run yarn test indicator or check the story "Indicator: with popover" of the storybook.

## Screenshots

[Gravação de tela de 14-06-2023 15:31:54.webm](https://github.com/Brisanet/ion/assets/98463818/ebe872dc-c579-45bf-8b1c-205f62293be2)


## View Storybook

[Storybook](https://62eab350a45bdb0a5818520e-ohlrmgtcme.chromatic.com/?path=/story/ion-data-display-indicator--with-popover)

## Compliance
